src/controllers/userController.js

```diff
@@ -1,62 +1,59 @@
 export default class UserController {
-    constructor(usersList) {
+    constructor(usersList, suggestedList) {
         this.usersList = usersList;
-    }
-    findUser(userId) {
-        return this.usersList.find((user) => user.id === userId);
+        this.suggestedList = suggestedList;
     }
     getUser(req, res) {
-        const currentUser = this.findUser(req.params.id);
-        if (currentUser) {
-            res.status(200).json(currentUser);
-        } else {
-            res.status(404).send('User not found');
-        }
+        this.usersList
+            .findOne({ where: { id: req.params.id } })
+            .then((user) => {
+                if (user) {
+                    res.status(200).json(user);
```

Comment:
@lenon777, just imagine that you need to write a small frontend application for your backend service.
In case when user exists in the database you receive user record JSON formatted, but if the user does not exist you receive string message...
How do you think will this backend service mature and easy to work with?